styles.css

```diff
@@ -0,0 +1,120 @@
+@import url('https://fonts.googleapis.com/css2?family=Crete+Round&family=Roboto+Serif:opsz,wght@8..144,100&display=swap');
+
+@media screen and (max-width: 640px) {
+  main {
+    padding: 0;
+    margin: 0;
+  }
+
+  header,
+  section {
+    margin: 0;
+    padding: 20px;
+    color: #fff;
+  }
+
+  header {
+    background: #3e3c3c;
+    display: flex;
+    justify-content: space-between;
+  }
+
+  #humberger {
+    display: none;
+  }
+
+  header a {
+    font-family: 'Roboto Serif', serif;
+    text-decoration: none;
+    font-weight: bold;
+    font-size: large;
+    color: #fff;
+    padding-left: 2%;
+  }
+
+  #navig {
+    height: fit-content;
+  }
+
+  #hum {
+    font-family: 'Roboto Serif', serif;
+    color: #fff;
+    font-size: 3vh;
+    background-color: #3e3c3c;
+    border: none;
+  }
+
+  #navitems {
+    display: none;
+  }
+
+  #navitems a {
+    box-sizing: border-box;
+    display: block;
+    width: 100%;
+    text-decoration: none;
+    padding-top: 5px;
+    color: #fff;
+    font-weight: 200;
+  }
+
+  #navitems a:hover {
+    background-color: aqua;
+    text-size-adjust: 10%;
+  }
+
+  #toolbar {
+    background-color: #1c1a19;
+    padding-left: 5vh;
+    background-image: url("assets/mobilegroup.svg");
+    background-repeat: no-repeat;
+    background-attachment: fixed;
+    background-position: bottom right;
+    background-position-x: 98%;
+    background-position-y: 70%;
+  }
+
+  #heytext {
+    font-family: 'Crete Round', serif;
+    color: #ff6b00;
+  }
```

Comment:
- As requested by the previous reviewer, Kindly make sure to adjust your title `font-size` to look like how it looks on the provided Figma design. :wink: 

   - ### Your design
   ![image](https://user-images.githubusercontent.com/87197412/186707928-48fbef46-e412-4b89-8085-1b81fed00b62.png)


  -  ### Figma design
   ![image](https://user-images.githubusercontent.com/87197412/186708008-b3a7b299-1b6d-4868-8f32-442b27bac55c.png)